packages/backend/src/app/ee/platform/platform.controller.ts

```diff
@@ -0,0 +1,37 @@
+import { FastifyPluginCallbackTypebox, Type } from '@fastify/type-provider-typebox'
+import { UpdatePlatformRequestBody } from '@activepieces/ee-shared'
+import { ApId } from '@activepieces/shared'
+import { platformService } from './platform.service'
+
+export const platformController: FastifyPluginCallbackTypebox = (app, _opts, done) => {
+    app.patch('/:id', UpdatePlatformRequest, async (req) => {
```

Comment:
`patch` and `post` is both valid, usually patch is used when you update certain fields and keeping others same, while post doesn't have this restriction in the convention it can either be means replace or update certian fields but to stay consistent with other APIs let's keep it post.